app/src/main/java/ar/com/wolox/android/example/ui/LoginActivity.java

```diff
@@ -0,0 +1,18 @@
+package ar.com.wolox.android.example.ui;
```

Comment:
Let's add this inside a package named `login`. This will give us a better organization for the future.